src/pages/ContactPage.vue

```diff
@@ -1,12 +1,93 @@
 <template>
-  <MainTitle text="This is Contact Page" />
+<div class="page-wrapper"> 
+   <ActionBar name="Contact us"/>
+   <header>
+      <MainTitle text="Contact us" positionType="left"/>
+   </header>
+  
+   <div class="main-contact">
+     <div class="main-contact__map"> </div>
+    
+     <div class="main-contact__info">
+       <ContactInfo/>
+     </div>
+
+     <div class="main-contact__links"> </div>
+
+  
+        
+   </div>
+</div>
+ 
 </template>
 
 <script>
 import MainTitle from "../components/MainTitle.vue";
+import ActionBar from '../components/ActionBar.vue';
+import ContactInfo from '../components/ContactInfo.vue';
 export default {
-  components: { MainTitle },
+  components: { MainTitle, ActionBar, ContactInfo },
 };
 </script>
 
+<style lang="scss">
+  .page-wrapper{
+    @include pageCenter()
+   }
+
+
+  .main-contact{
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    width: 327px;
+    margin: 20px auto;
+    background-color: $cl-athens-grey;
+
+    &__map{
+      width: 327px;
+      height: 284px;
```

Comment:
Setting width and height as fixed pixels don't seem like a good idea here. As far as I know, the map takes like 50% of the layout, why not make it `width: 50%`?

Reply:
This line refers to media query for mobile - the width of map should be 100% here - I changed to 100%;
For the media query: min-width 1015px I changed to 50% - as you suggest.